cmd/packer-plugins-check/main.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/hashicorp/packer/packer/plugin"
 )
 
-const packerPluginCheck = "packer-plugin-check"
+const packerPluginsCheck = "packer-plugins-check"
```

Comment:
I had to change this name because packer was thinking this was a plugin 🤦🏼

Reply:
oh right !